content/influxdb/cloud/organizations/buckets/update-bucket.md

```diff
@@ -23,24 +23,25 @@ If you change a bucket name, be sure to update the bucket in the above places as
 
 ## Update a bucket's name in the InfluxDB UI
 
-1. In the navigation menu on the left, select **Data (Load Data)** > **Buckets**.
+1. In the navigation menu on the left, select **Data ** > **Buckets**.
 
-    {{< nav-icon "data" >}}
+    {{![image](https://user-images.githubusercontent.com/9125473/109828439-3e2f5400-7c02-11eb-8fb3-f0e8b5f34a3a.png)}}
```

Comment:
This is a convention we've established to address both the collapsed and expanded states of the left nav. These should not be changed.